ascon-aead/src/lib.rs

```diff
@@ -0,0 +1,290 @@
+// Copyright 2021-2022 Sebastian Ramacher
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#![no_std]
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
We're migrating to `doc_auto_cfg` FWIW (not sure it's really relevant here either way)

Reply:
Fixed.